projects/plugins/jetpack/class.jetpack.php

```diff
@@ -3332,6 +3332,9 @@ public function admin_init() {
 		/** Already documented in automattic/jetpack-connection::src/class-client.php */
 		$client_verify_ssl_certs = apply_filters( 'jetpack_client_verify_ssl_certs', false );
 
+		// Run post-activation actions if needed.
+		$this->plugin_post_activation();
```

Comment:
do we need to wait for the admin to load in order to call `wp_safe_redirect`? 
Would it make sense to just redirect in the plugin activation hook?

Reply:
Discussed further in Slack: p1707156725034619-slack-C01264051NE